docs/en/sql-reference/distributed-ddl.md

```diff
@@ -19,4 +19,8 @@ The local version of the query will eventually be executed on each host in the c
 
 :::warning    
 The order for executing queries within a single host is guaranteed.
-:::
\ No newline at end of file
+:::
+
+:::warning    
+There is a special process inside ClickHouse server -- DDLWorker. It starts when ClickHouse starts. When you execute some SQL with `on cluster` ClickHouse creates tasks in ZooKeeper for each shard in the cluster. DDLWorker on each node takes its own tasks and executes them locally. But because DDLWorker starts with ClickHouse it uses settings from the default profile which were defined when ClickHouse started. There is no way to restart DDLWorker. To apply changes in profile settings you need to restart ClickHouse.
```

Comment:
It's true, but with `distributed_ddl_entry_format_version = 2` session/query settings are passed from initiator to all hosts